src/components/inputs/z-combobox/index.tsx

```diff
@@ -147,6 +138,93 @@ export class ZCombobox {
     this.selectedCounter = this.itemsList.filter((item) => item.checked).length;
   }
 
+  private getControlToListSize(): ListSize {
+    let size;
+    switch (this.size) {
+      case ControlSize.BIG:
+        size = ListSize.LARGE;
+        break;
+      case ControlSize.SMALL:
+        size = ListSize.SMALL;
+        break;
+      case ControlSize.X_SMALL:
+        size = ListSize.SMALL;
+        break;
+      default:
+        size = ListSize.MEDIUM;
+    }
+
+    return size;
+  }
+
+  private getItemId(item: ComboItem): string {
+    return `combo-checkbox-${this.inputid}-${item.id}`;
+  }
+
+  private getCheckAllId(): string {
+    return `combo-checkbox-${this.inputid}-check-all`;
+  }
+
+  private resetInputTabIndex(): void {
```

Comment:
Ma non possiamo dare una classe a tutte ste checkbox e cavarcela con un selettore unico, tipo 
`this.element.shadowRoot.querySelector(`.classe-di-tutte-le-checkbox-di-${this.htmlid}`).setAttribute("tabindex", "-1");`